ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/batch/BatchedSubmissionValidatorFactory.scala

```diff
@@ -69,48 +65,25 @@ object BatchedSubmissionValidatorFactory {
       stateCache: Cache[DamlStateKey, DamlStateValue],
       cacheUpdatePolicy: CacheUpdatePolicy,
       keySerializationStrategy: StateKeySerializationStrategy = DefaultStateKeySerializationStrategy,
-      ledgerDataExporter: LedgerDataExporter = LedgerDataExporter())(
-      implicit executionContext: ExecutionContext)
+  )(implicit executionContext: ExecutionContext)
     : (DamlLedgerStateReader with QueryableReadSet, CommitStrategy[LogResult]) = {
     val ledgerStateReader = new CachingDamlLedgerStateReader(
       stateCache,
       cacheUpdatePolicy.shouldCacheOnRead,
       keySerializationStrategy,
       DamlLedgerStateReader.from(
         new LedgerStateReaderAdapter[LogResult](ledgerStateOperations),
-        keySerializationStrategy)
+        keySerializationStrategy,
+      ),
     )
     val commitStrategy = new CachingCommitStrategy(
       stateCache,
       cacheUpdatePolicy.shouldCacheOnWrite,
       new LogAppendingCommitStrategy[LogResult](
         ledgerStateOperations,
         keySerializationStrategy,
-        ledgerDataExporter)
+      )
     )
     (ledgerStateReader, commitStrategy)
   }
-
-  case class CachingEnabledComponents[LogResult](
```

Comment:
Aha, sorry, I thought it was unused. I will revert it.

Reply:
Turns out it is unused. 🙂 

BTW, I deleted it because the alternative was to push the `ledgerDataExporter` property into the parameters list, which seemed like an unnecessary change given that it's, well, not used anywhere.